AudioKit/Common/Sequencing/Apple Sequencer/MIDIMetaEvent+allocate.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKit/
+
+import Foundation
+
+extension MIDIMetaEvent {
+    /// `MIDIMetaEvent` is a variable length C structure. YOU MUST create one using this function
```

Comment:
I read your comment 'YOU MUST' literally... anyway we can lock it down so there is no possible way to do this otherwise?
I create a lot of metaEvents in the file reader - will those just all crash now?